components/AboutInfo.js

```diff
@@ -0,0 +1,15 @@
+import styles from './css/AboutInfo.module.css'
+
+const AboutInfo = () => {
+    return (
+        <div className={styles.about}>
+            <h1>About</h1>
+            <p>
+                This site was created to help new developers contribute to an open source website. Please contact <a href="https://github.com/SamSamskies">Sam</a> for an invitation to contribute and for any 
+                pull requests to be added.
+            </p>
+        </div>
+    )
+}
+
+export default AboutInfo
```

Comment:
What do you think about moving this code into the `About` component? This is pretty simple. Doesn't seem necessary to extract it out into a separate component.

Reply:
What does the prettier do and does this get install through yarn?

Reply:
Prettier is a code formatter widely used in the JavaScript community https://prettier.io/. You should already have it installed since it's a dev dependency in this project. Try running `yarn format` and see what happens. I configure my IDE to format on save https://prettier.io/docs/en/editors.html.

Reply:
Ok I used it and it did update the code formart. Thanks for clarifying! Its addressed on the latest pull request.